src/isomorphic/hooks/ReactComponentTreeHook.js

```diff
@@ -39,6 +43,24 @@ function isNative(fn) {
   }
 }
 
+type Item = {
+  element: ReactElement,
+  parentID: DebugID,
+  text: ?string,
+  childIDs: Array<DebugID>,
+  isMounted: bool,
+  updateCount: number,
+};
+
+declare function getItem(id: DebugID): ?Item;
```

Comment:
Is this going to result in a global definition of getItem or is it local to this module? I thought these weren't supposed to be used in local code.